config/config.go

```diff
@@ -177,7 +181,13 @@ func (o *Optimus) GetJob() Job {
 
 func (o *Optimus) GetDatastore() []Datastore {
 	ds := []Datastore{}
-	_ = o.k.Unmarshal("datastore", &ds)
+	if o.k.Get("datastore") != nil {
```

Comment:
I have tested by using env var, however OPTIMUS_DATASTORE, OPTIMUS_DATASTORE_PATH, OPTIMUS_DATASTORE_TYPE, and OPTIMUS_DATASTORE_BACKUP are not working. Looks like Koanf unmarshal is error for this as Koanf returns all the value in string.

Therefore I catch the error and do json unmarshal instead for cases like this. Please check this part as well @kushsharma 🙂